app/scripts/lib/personal-message-manager.js

```diff
@@ -241,6 +242,14 @@ export default class PersonalMessageManager extends EventEmitter {
     this._setMsgStatus(msgId, 'rejected');
   }
 
+  /**
+   * Clears all unapproved messages from memory.
+   */
+  clearUnapproved() {
+    this.messages = [];
```

Comment:
This seems to clear _all_ messages :thinking: Not just unapproved messages.

Not sure I'm reading this right or not, but it looks like in prod right now we keep messages indefinitely, but they don't get persisted so they're lost upon restart.

Reply:
Since we never do anything with approved messages, and since they get cleared on restart, clearing them here seems fine :man_shrugging: 

I guess we should update the method name and description though.

Reply:
I decided to just filter out the unapproved messages here instead of resetting them completely. the function signature and description now align with the code.